src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java

```diff
@@ -0,0 +1,206 @@
+package com.wikia.webdriver.testcases.seotests;
+
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.configuration.Configuration;
+import com.wikia.webdriver.common.properties.Credentials;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+
+import org.testng.annotations.DataProvider;
+import org.testng.annotations.Test;
+
+/**
+ * @ownership Seo
+ */
+public class HtmlTitleTests extends NewTestTemplate {
+
+  Credentials credentials = Configuration.getCredentials();
+
+  @DataProvider
+  private Object[][] dataHtmlTitleTest001() {
+    return new Object[][]{
+        // original title
+        {
+            "sktest123",
```

Comment:
But why? And how do you name it?

String WIKI_ADTEST = "adtest"

I think "adtest" is more explicit.

On 10/05/2015 07:57 AM, Dmitry Rets wrote:
> In
> src/test/java/com/wikia/webdriver/testcases/seotests/HtmlTitleTests.java
> <https://github.com/Wikia/selenium-tests/pull/885#discussion_r41153353>:
>
>> +import org.testng.annotations.DataProvider;
>> +import org.testng.annotations.Test;
>> +
>> +/**
>> + * @ownership Seo
>> + */
>> +public class HtmlTitleTests extends NewTestTemplate {
>> +
>> +  Credentials credentials = Configuration.getCredentials();
>> +
>> +  @DataProvider
>> +  private Object[][] dataHtmlTitleTest001() {
>> +    return new Object[][]{
>> +        // original title
>> +        {
>> +            "sktest123",
>
> @gabrys <https://github.com/gabrys> actually, you are right :+1:
> What do you think about combining to variable the same string literals
> from few dataProviders?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Wikia/selenium-tests/pull/885/files#r41153353>.
>